client/python/test/test_client.py

```diff
@@ -665,6 +666,39 @@ def test_fetch_old_version_space(self):
             "unnamed_endpoints": {},
         }
 
+    def test_unnamed_endpoints_use_fn_index(self):
+
+        with gr.Blocks() as demo:
+            name = gr.Textbox()
+            greeting = gr.Textbox()
+            greet = gr.Button()
+            greet.click(lambda x: f"Hello {x}", name, greeting)
+
+        _, local_url, _ = demo.launch(prevent_thread_lock=True)
+
+        expected = textwrap.dedent(
+            """Client.predict() Usage Info
+---------------------------
+Named API endpoints: 0
+
+Unnamed API endpoints: 1
+
+ - predict(parameter1, fn_index=0) -> value2
```

Comment:
Suggestion, maybe just check if `"fn_index=0"` is in the string, as opposed to `"api_name='0'"` as the format of the `view_api(return_format="str")` may change and we don't want this test to be welded too tightly to the format

Reply:
Fixed now @abidlabs ! Thanks